### PR TITLE
Fix InstalledDesktopApp.GetInventoryAsync example

### PR DESCRIPTION
**Pull request by sylveon**

### CODE REVIEW

**Comment by PRMerger8** [comment]
@sylveon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@sylveon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: windows.system.inventory/installeddesktopapp.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ InstalledDesktopApp checks for the **packageQuery** capability and returns an em
 
 ## -examples
 ```csharp
-IList<IInstalledDesktopApp> installedApps = await InstalledDesktopApp.GetInventoryAsync();
+IReadOnlyList<IInstalledDesktopApp> installedApps = await InstalledDesktopApp.GetInventoryAsync();
```
Comment (asklar) [inline windows.system.inventory/installeddesktopapp.md:22]: since the type isn't important, consider just using `var` here and below

Comment (sylveon) [reply windows.system.inventory/installeddesktopapp.md:22]: Possible. Is there any convention regarding the use of `var` in documentation?

**Comment by Jak-MS** [comment]
#label:"aq-pr-triaged"

**Pull request merged**